### PR TITLE
Feature/global ens wave

### PR DESCRIPTION
**Pull request by DeannaSpindler-NOAA**
This code has the addition of the lead_average plots, and has been running on my cron for months with no issues.  The only concern is that the stats file archive will have gaps (due to machine switchovers) since I have not been monitoring this project.

To run the prep step, make sure to modify the OUTPUTROOT directory under the "developers directories" in  ecf/global_ens/prep/jevs_global_ens_wave_grid2obs_prep.ecf  
and to point COMIN to either my stats file archive ‘/lfs/h2/emc/vpppg/noscrub/deanna.spindler/EVS_stats/evs/v1.0’ or to the parallel stats files (because mine will have gaps): ‘/lfs/h2/emc/vpppg/noscrub/emc.vpppg/evs/v1.0’  because the prep jobs use COMIN to access the stats file archive.  
Then: qsub jevs_global_ens_wave_grid2obs_prep.ecf 
The work directory will be under: /lfs/h2/emc/stmp/$USER/evs_output  and is named jevs_global_ens_wave_g2o_prep.{qsubID}
 The run will make ${OUTPUTROOT}/evs/v1.0/prep/global_ens/wave.YYYYMMDD/gefs/grid2obs  where you will find 5 NetCDF files and 260 grib2 files per day.

To run the stats step, also change the OUTPUTROOT directory in 
ecf/global_ens/stats/jevs_global_ens_wave_grid2obs_stats.ecf 
to the same directory as you did in the prep ecf,  then 
qsub jevs_global_ens_wave_grid2obs_stats.ecf 
The work directory will be under the same /lfs/h2/emc/stmp/$USER/evs_output but this time is named jevs_global_ens_wave_g2o_stats.{qsubID} 
This run will make ${OUTPUTROOT}/evs/v1.0/stats/global_ens/wave.YYYYMMDD/gefs/grid2obs  where you will find 260 small stat files: point_stat_fcstGEFS_obsGDAS_climoERA5_*.stat  and ${OUTPUTROOT}/evs/v1.0/stats/global_ens/gefs.YYYYMMDD/evs.stats.gefs.wave.grid2obs.vYYYYMMDD.stat

To run the plots step, change the OUTPUTROOT in
ecf/global_ens/plots/jevs_global_ens_wave_grid2obs_plots.ecf
to my OUTPUTROOT='/lfs/h2/emc/vpppg/noscrub/deanna.spindler/EVS_stats/evs/v1.0' or to the parallel stats files (because mine will have gaps): ‘/lfs/h2/emc/vpppg/noscrub/emc.vpppg/evs/v1.0’
because the job will need this to access the archive.
Then
qsub jevs_global_ens_wave_grid2obs_plots.ecf
The work directory will be under the same /lfs/h2/emc/stmp/$USER/evs_output
but this time is named jevs_global_ens_wave_g2o_plots.{qsubID}
This run will make ${OUTPUTROOT}/evs/v1.0/plots/global_ens/global_ens.YYYYMMDD
where you will find the two tar files:
evs.plots.global_ens.wave.grid2obs.last31days.vYYYYMMDD.tar
and
evs.plots.global_ens.wave.grid2obs.last90days.vYYYYMMDD.tar

### CODE REVIEW

**Comment by ShelleyMelchior-NOAA** [comment]
Just a note: backlog of prep data can be found in `/lfs/h2/emc/vpppg/noscrub/emc.vpppg/evs/v1.0/prep/global_ens/wave*`

**Comment by PerryShafran-NOAA** [comment]
Hi, @DeannaSpindler-NOAA ,

Just completed a test for this particular PR.  Please examine the output here:

prep: /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/prep/global_ens/wave.20230406
stats: /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/stats/global_ens/*20230406*
plots: /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/plots/global_ens/gefs.20230406

Thanks!

Perry

**Comment by DeannaSpindler-NOAA** [comment]
Hello Perry,

The test looks good, all the stat files and plots are there (including the lead average plots).

Thanks,
Deanna

File: crons/gefs_wave_evs.sh
Comment (PerryShafran-NOAA) [inline crons/gefs_wave_evs.sh:1]: What are these cron files for?

**Comment by DeannaSpindler-NOAA** [comment]
The crons run the EVS and push the graphics to the website.
I was asked to include them when I left EMC.

**Comment by PerryShafran-NOAA** [comment]
You mean to include them in the authoritative repository?

**Comment by DeannaSpindler-NOAA** [comment]
That I do not know.  I wouldn't think they'd be included, I think it was mostly left for whoever takes over.

**Comment by PerryShafran-NOAA** [comment]
Certainly we'll need to save the crons on github for the next person.  However, I don't think they belong in the EVS repository as they won't be part of ops.  @AliciaBentley-NOAA @ShelleyMelchior-NOAA I think we should remove the crons directory from the repository, but be saved elsewhere on github for the next person.

**Comment by AliciaBentley-NOAA** [comment]
Hi Perry,

I agree. We don't want to get rid of the cronjobs entirely (they should be
in Deanna's space on Github somewhere), but the cronjobs don't need to be
in the EVS authoritative repository because they won't be delivered to
NCO.

Take care,
Alicia
___________________________________
*Alicia M. Bentley, Ph.D.*
Global Verification Project Lead
Model Evaluation Group (MEG) <https://www.emc.ncep.noaa.gov/users/meg/home/>
NOAA/NWS/NCEP/EMC (VPPPG Branch)
Work Phone: ‪(301) 683-3783‬

**Comment by ShelleyMelchior-NOAA** [comment]
@DeannaSpindler-NOAA, I recommend you create a new repository in your DeannaSpinder-NOAA account that contains your EVS crons for your EVS components. That way your crons can be easily referenced by the incoming team member to pick up the running of them as she becomes versed and has WCOSS2 access.

**Comment by PerryShafran-NOAA** [comment]
In the meantime, @DeannaSpindler-NOAA, please remove the crons directory from the EVS repository, and then I can approve and merge the code.

Perry

**Comment by DeannaSpindler-NOAA** [comment]
Hello Perry,
The cron directories have been removed from the EVS repos, and commited to the separate EVS_crons repository.
Best,
Deanna

**Comment by PerryShafran-NOAA** [comment]
Thank you!  I'll approve this one, then start to work on the other two, which hopefully should be quick.

Perry

**Review by PerryShafran-NOAA — APPROVED** [review]

**Pull request merged**